data/hai/hai missions.txt

```diff
@@ -1932,75 +1932,101 @@ mission "Station Exploration Abandon Ship"
+			branch pendant
+				has "outfit: Shield Beetle Pendant"
+			branch normal
+				has "outfit: Quantum Keystone"
+			branch remnant
+				has "outfit: Quantum Key Stone"
+			branch huh
+				has "outfit: Tree Skeleton Key Stone"
+			label normal
+			`	"I see that you are wiser than most humans here," he says, pointing towards your ship. Its hatch is open, prominently displaying your Keystone to any potential travellers. "You carry a shield against the tempest of space. Let no one else tell you otherwise." He stops to catch his breath. "Now, onto more worldly things. My name is Hareek. Are you willing and able to take me to <destination> for <payment>?"`
+				goto choice
+			label remnant
+			`	"The keystone you carry is a strange one, human," he says, pointing towards your ship. Its hatch is open, prominently displaying your Keystone to any potential travellers. He walks up to it to get a closer look. "Very unusual. No proper mounting either - must be a human design." He sighs. "Well, at least the Keystone itself looks authentic. It will have to do." He turns to you. "You carry a shield against the tempest of space. Let no one else tell you otherwise. If your people have access to Keystones as well, then I hope they too become wise to its power, and learn to treat them with more reverence. Now, onto more worldly things. My name is Hareek. Are you willing and able to take me to <destination> for <payment>?"`
+				goto choice
+			label pendant
+			`	"A unique design you have here, Captain," he says, pointing towards your ship. Its hatch is open, prominently displaying Nanachi's pendant to any potential travellers. "A gift, I presume? You are lucky you have someone looking out for you. You carry a shield against the tempest of space. Let no one else tell you otherwise." He stops to catch his breath. "Now, onto more worldly things. My name is Hareek. Are you willing and able to take me to <destination> for <payment>?"`
+				goto choice
+			label huh
+			`	"I've never seen anything like this... thing you have," he says, pointing towards your ship. Its hatch is open, prominently displaying the Tree Skeleton Key Stone to any potential travellers. The Hai sounds more than a little disgusted. "I hope you haven't done anything you'll regret."`
+			`	He takes a closer look. "It seems to be made out of the same material as a Keystone, and yet it is entirely foreign to me. Where you got this, I have no idea. Well, at least the material itself looks authentic. It will have to do." He sighs and turns to you. "I suspect that whoever you got this from is not happy it is out of their hands. I would show more respect for Keystones if I were you." He stops to catch his breath. "Now, onto more worldly things. My name is Hareek. Are you willing and able to take me to <destination> for <payment>?"`
```

Comment:
```suggestion
			`	He takes a closer look. "It seems to be made out of the same material as a keystone, and yet it is entirely foreign to me. Where you got this, I have no idea. Well, at least the material itself looks authentic. It will have to do." He sighs and turns to you. "I suspect that whoever you got this from is not happy it is out of their hands. I would show more respect for keystones if I were you." He pauses to take a breath. "Now, onto more worldly things. My name is Hareek. Are you willing and able to take me to <destination> for <payment>?"`
```